packages/payments-service/src/core/router.ts

```diff
@@ -0,0 +1,13 @@
+import { AppRequest, AppResponse } from '@reapit/node-utils'
+import { Router } from 'express'
+import { createApiKey } from '../controllers/create-api-key'
+import { getPayment } from '../controllers/get-payment'
+import { updatePayment } from '../controllers/update-payment'
+
+const router = Router()
+
+router.post('/api-key', (req: AppRequest, res: AppResponse) => createApiKey(req, res))
```

Comment:
You can do also do it like this if you wanted - I find a bit simpler to read but you might disagree!

`router.post('/api-key', createApiKey)`